buildSrc/build.gradle.kts

```diff
@@ -9,10 +13,14 @@ repositories {
 }
 
 dependencies {
-    implementation(kotlin("gradle-plugin", version = "1.7.0"))
-    implementation(kotlin("serialization", version = "1.6.21"))
-    implementation("org.jetbrains.dokka", "dokka-gradle-plugin", "1.6.21")
-    implementation("org.jetbrains.kotlinx", "atomicfu-gradle-plugin", "0.17.3")
+    implementation(kotlin("gradle-plugin"))
+    implementation(kotlin("serialization"))
+    implementation("org.jetbrains.dokka", "dokka-gradle-plugin", "1.7.0")
+    implementation("org.jetbrains.kotlinx", "atomicfu-gradle-plugin", "0.18.2")
     implementation(gradleApi())
     implementation(localGroovy())
 }
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.languageVersion = "1.5"
+}
```

Comment:
Do we need this? Everything needs to work fine without it.

Reply:
That should be set to 1.7, if you do keep it. If I remember correctly there is something it does, but I don't fill remember

Reply:
languageVersion | Provide source compatibility with the specified version of Kotlin | "1.4" (DEPRECATED), "1.5", "1.6", "1.7"
-- | -- | --


I think it should stay on 1.5